python/pysrc/kaskada/_timestream.py

```diff
@@ -264,6 +264,19 @@ def floor(self) -> Timestream:
         """Return a Timestream of self rounded down to the nearest integer."""
         return Timestream._call("floor", self)
 
+    def hash(self) -> Timestream:
+        """Return a Timestream containing the hash of the input.
+
+        Notes:
+            Unlike many functions which return `null` if an input is `null`,
```

Comment:
The note reads weird. Maybe:

> `hash` will only return `null` when interpolated at points the input is not defined. At other points, it will return the hash of `null`, which is not `null`.

Reply:
I copied the node directly from the old docs.  but with the new implementation of `hash`, I'm going to change it to: 

> `hash` will only return `null` when interpolated at points the input is not defined. At other points, it will return the hash of `null`, which is `0` (not `null`).